// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`EditGraphQLError > renders a CloseEvent correctly 1`] = `
"GGT_CLI_CLIENT_ERROR: An error occurred while communicating with Gadget

The connection to Gadget closed unexpectedly.

If you think this is a bug, please submit an issue using the link below.

https://github.com/gadget-inc/ggt/issues/new?template=bug_report.yml&error-id=00000000-0000-0000-0000-000000000000"
`;

exports[`EditGraphQLError > renders a GraphQL error correctly 1`] = `
"GGT_CLI_CLIENT_ERROR: An error occurred while communicating with Gadget

Gadget responded with the following error:

  Changed and deleted files must not overlap

If you think this is a bug, please submit an issue using the link below.

https://github.com/gadget-inc/ggt/issues/new?template=bug_report.yml&error-id=00000000-0000-0000-0000-000000000000"
`;

exports[`EditGraphQLError > renders a string correctly 1`] = `
"GGT_CLI_CLIENT_ERROR: An error occurred while communicating with Gadget

We received a response without data

If you think this is a bug, please submit an issue using the link below.

https://github.com/gadget-inc/ggt/issues/new?template=bug_report.yml&error-id=00000000-0000-0000-0000-000000000000"
`;

exports[`EditGraphQLError > renders an ErrorEvent correctly 1`] = `
"GGT_CLI_CLIENT_ERROR: An error occurred while communicating with Gadget

connect ECONNREFUSED 10.254.254.254:3000

If you think this is a bug, please submit an issue using the link below.

https://github.com/gadget-inc/ggt/issues/new?template=bug_report.yml&error-id=00000000-0000-0000-0000-000000000000"
`;

exports[`EditGraphQLError > renders multiple GraphQL errors correctly 1`] = `
"GGT_CLI_CLIENT_ERROR: An error occurred while communicating with Gadget

Gadget responded with multiple errors:

  1. Changed and deleted files must not overlap
  2. Files version mismatch, expected 1 but got 2

If you think this is a bug, please submit an issue using the link below.

https://github.com/gadget-inc/ggt/issues/new?template=bug_report.yml&error-id=00000000-0000-0000-0000-000000000000"
`;

exports[`InvalidSyncFileError > renders correctly 1`] = `
"GGT_CLI_INVALID_SYNC_FILE: The .gadget/sync.json file was invalid or not found

We failed to read the Gadget metadata file in this directory:

  ~/gadget/test

If you're running \`ggt sync\` for the first time, we recommend using an empty directory such as:

  ~/gadget/test

Otherwise, if you're sure you want to sync the contents of that directory to Gadget, run \`ggt sync\` again with the \`--force\` flag:

  $ ggt sync --app test ~/gadget/test --force

You will be prompted to either merge your local files with your remote ones or reset your local files to your remote ones.

If you think this is a bug, please submit an issue using the link below.

https://github.com/gadget-inc/ggt/issues/new?template=bug_report.yml&error-id=00000000-0000-0000-0000-000000000000"
`;

exports[`UnexpectedError > renders correctly 1`] = `
"GGT_CLI_UNEXPECTED_ERROR: An unexpected error occurred

Error: Whoops!
    at <anonymous>:1:1

This is a bug, please submit an issue using the link below.

https://github.com/gadget-inc/ggt/issues/new?template=bug_report.yml&error-id=00000000-0000-0000-0000-000000000000"
`;

exports[`YarnNotFoundError > renders correctly 1`] = `
"GGT_CLI_YARN_NOT_FOUND: Yarn not found

Yarn must be installed to sync your application. You can install it by running:

  $ npm install --global yarn

For more information, see: https://classic.yarnpkg.com/en/docs/install"
`;
